templates/scripts/pre-k8s.sh

```diff
@@ -11,7 +11,7 @@ OATHKEEPER_SECRET_NAME=${PROJECT}-${ENVIRONMENT}-oathkeeper-jwks-${RANDOM_SEED}
 aws secretsmanager describe-secret --region ${AWS_DEFAULT_REGION} --secret-id ${VPN_SECRET_NAME} > /dev/null 2>&1
 if [[ $? -ne 0 ]]; then
     echo "Creating VPN private key..."
-    secret=$(kubectl run --rm --quiet --attach=true --context ${KUBE_CONTEXT} zero-k8s-utilities --image=commitdev/zero-k8s-utilities:${ZERO_K8S_UTILS_VERSION} --restart=Never -- wg genkey)
+    secret=$(docker run commitdev/zero-k8s-utilities:${ZERO_K8S_UTILS_VERSION} "sh" "-c"  "wg genkey")
```

Comment:
Did you update the prerequisites to include `docker`? Or is that assumed with `kubectl`?

Reply:
Yeah we are going to have to update the dependencies in the zero binary repo.